src/RecordManager/Base/Enrichment/OnkiLightEnrichment.php

```diff
@@ -352,6 +383,82 @@ function ($s) {
         }
     }
 
+    /**
+     * Process WKT location data and add that information to solrArray
+     *
+     * Implementation notes
+     * Currently supports only single, non-empty 2D WKT point data format
+     *
+     * @param string $wkt       Well-known text to be processed
+     * @param array  $solrArray Metadata to be sent to Solr
+     *
+     * @return void
+     */
+    protected function processLocationWkt($wkt, &$solrArray): void
+    {
+        preg_match(
+            '/^POINT\s*\((?P<lon>-?\d+(?:\.?\d+)?)\s*(?P<lat>-?\d+(?:\.?\d+)?)\)/',
+            $wkt,
+            $matches
+        );
+        if ($matches) {
+            $wktItem = [
+                'lat' => $matches['lat'],
+                'lon' => $matches['lon'],
+                'wkt' => 'POINT(' . $matches['lon'] . ' ' . $matches['lat'] . ')'
```

Comment:
This one here is only for normalization, otherwise one could just simply set $wkt there.